### DISCUSSION TITLE
How can I downland old version Homebrew in my old mac

### DISCUSSION THREAD

**Discussion by Msoobe** [Homebrew/discussions#5165] [Getting started]
### Output of `brew config`

```shell
You MacOS version is too old
```

### Output of `brew doctor`

```shell
You MacOS version is too old
```

### Description of issue

my macos version is 10.9.5，when I run command in brew.sh it tell me my macos version is too old.
but I don't like upgrade my mac or setup linux,how can I download old version Homebrew?

**Comment by SMillerDev** [comment]
You can not. Homebrew will always automatically update to the latest version, which will tell you your Mac is too old again.

**Reply by Msoobe** [reply]
Thank you tell me about this.

**Comment by Msoobe** [comment]
这是来自QQ邮箱の自动回复邮件。Oh，shit.

**Comment by gromgit** [comment]
> my macos version is 10.9.5，when I run command in brew.sh it tell me my macos version is too old.

For a macOS that old, you might want to consider [MacPorts](https://www.macports.org/) instead. Installation is [pretty straightforward](https://www.macports.org/install.php), though you must have admin access (i.e. be able to `sudo`).

**Reply by Msoobe** [reply]
ok,I will try it.Thank u.

**Reply by kencu** [reply]
you can install macports in your home directory without admin privs or need for sudo, if that is important for you:

https://superuser.com/questions/814422/can-you-install-macports-to-the-home-directory-and-still-have-everything-work

The default installation does require sudo / admin privs to install software that is used globally to a common directory, true.
